rule.php

```diff
@@ -86,22 +86,19 @@ public static function add_settings_form_fields(
                     0 => get_string('notrequired', 'quizaccess_honestycheck'),
                     1 => get_string('honestycheckrequiredoption', 'quizaccess_honestycheck'),
                 ));
+        $default = get_config('quizaccess_honestycheck', 'honestycheckrequired');
+        $mform->setDefault('honestycheckrequired', $default);
```

Comment:
What is the point of the lcoal variable $default here? Much better to put the $default inline in the next line.